src/nvim/keymap.c

```diff
@@ -573,11 +571,7 @@ int find_special_key(const char_u **srcp, const size_t src_len, int *const modp,
     if (*bp == '-') {
       last_dash = bp;
       if (bp + 1 <= end) {
-        if (has_mbyte) {
-          l = mb_ptr2len_len(bp + 1, (int) (end - bp) + 1);
-        } else {
-          l = 1;
-        }
+        l = utfc_ptr2len_len(bp + 1, (int)(end - bp) + 1);
```

Comment:
I think we already agreed to keep the `mb_` names. `utf_` will be removed.

Reply:
It is very bad idea, `mb_…` references *both* `utf` and `utfc` and they have different enough semantics. So I never use this variant.